inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,139 @@
+#Pseudocode may look like actual code...please ignore...
+getPdbIds <- function(targetSeq, startPos = 1 , endPos = nchar(targetSeq), convert = TRUE, silent = TRUE){
+  
+  #get packages
+
+  library(bio3d)
+  library(RCurl)
+  library(XML)
+  library(seqinr)
+  
```

Comment:
I see that I need to add my packages onto the description but I thought you were requesting that I put a line into the module checking for a valid input. Was this not correct?  If it was, I don't see how these links answer my question - they don't contain information about a standard of whether or not to return Null or raise errors upon invalid input.